src/js/angular/sparql-editor/controllers.js

```diff
@@ -300,6 +292,78 @@ function SparqlEditorCtrl($scope, $location, $jwtAuth, $repositories, toastr, $t
     // ================================
     // =     Setup output handlers    =
     // ================================
+
+    /**
+     * Handles the "query" event emitted by the ontotext-yasgui. The event is fired immediately before sending the
+     * request and the request object can be altered here, and it will be sent with these changes.
+     * @param {object} queryResponseEvent - the event payload containing the query and the request object.
+     */
+    function queryHandler(queryResponseEvent) {
```

Comment:
Please change all these new functions to function expressions. See the other functions in the controller.